lib/mayaUsd/ufe/UsdAttribute.h

```diff
@@ -272,5 +342,32 @@ class UsdAttributeDouble3 : public TypedUsdAttribute<Ufe::Vector3d>
     create(const UsdSceneItem::Ptr& item, const PXR_NS::UsdAttribute& usdAttr);
 }; // UsdAttributeDouble3
 
+#if (UFE_PREVIEW_VERSION_NUM >= 4015)
+//! \brief Interface for USD Matrix3d (double) attributes.
+class UsdAttributeMatrix3d : public TypedUsdAttribute<Ufe::Matrix3d>
+{
+public:
+    typedef std::shared_ptr<UsdAttributeMatrix3d> Ptr;
+
+    using TypedUsdAttribute<Ufe::Matrix3d>::TypedUsdAttribute;
+
+    //! Create a UsdAttributeDouble3.
```

Comment:
Comment should be UsdAttributeMatrix3d

Reply:
This comment does not provide any extra information. Removed since reading the function signature is sufficient.